spec/vcr_helper.rb

```diff
@@ -16,8 +17,12 @@
 
   # Ignore requests other than to the API endpoints and LAA fee calculator
```

Comment:
Should probably update to
```ruby
#Ignore requests other than to the API endpoints, LAA fee calculator and google maps
```
for clarity

Reply:
👍  will do..